src/activity_actor.cpp

```diff
@@ -94,15 +94,15 @@ aim_activity_actor aim_activity_actor::use_bionic( const item &fake_gun,
     aim_activity_actor act = aim_activity_actor();
     act.weapon_source = WeaponSource::Bionic;
     act.bp_cost_per_shot = cost_per_shot;
-    act.fake_weapon = shared_ptr_fast<item>( new item( fake_gun ) );
+    act.fake_weapon = std::make_shared<item>( fake_gun );
```

Comment:
```suggestion
    act.fake_weapon = make_shared_fast<item>( fake_gun );
```
And the same for other changes in the last commit.